### PR TITLE
IRIS-1620 | Hide open graph when there is content image

### PR DESCRIPTION
**Pull request by rogatty**
## Links

* https://wikia-inc.atlassian.net/browse/IRIS-1620

## Description

A content image has priority over an open graph. Save both to service, it's handled by mobile apps as expected.

## Reviewers

@slayful

### CODE REVIEW

File: front/main/app/models/discussion/domain/content-images.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const DiscussionContentImages = Object.extend(
 		}),
 
 		hasImages() {
-			return this.get('images.length');
+			return this.get('images.length') > 0;
```
Comment (slayful) [inline front/main/app/models/discussion/domain/content-images.js]: Will this be fine if images are null?

File: front/main/app/mixins/discussion-editor-opengraph.js [hunk 2]
```diff
@@ -99,39 +116,31 @@ export default Ember.Mixin.create({
 	},
 
 	setOpenGraphProperties(url) {
-		if (this.get('showsOpenGraphCard')) {
+		if (this.get('openGraph') || this.get('contentImages').hasImages()) {
```
Comment (slayful) [inline front/main/app/mixins/discussion-editor-opengraph.js]: Perhaps extract to a separate method?
`hasOpenGraphOrContentImage()`?

File: front/main/app/models/discussion/domain/content-images.js (hunk 1 above)
Comment (rogatty) [reply front/main/app/models/discussion/domain/content-images.js]: They will never be null, see https://github.com/Wikia/mercury/blob/6c9272a3c736fcaae21cb81903d5263023119681/front/main/app/models/discussion/domain/content-images.js#L9-L16

File: front/main/app/mixins/discussion-editor-opengraph.js (hunk 2 above)
Comment (slayful) [reply front/main/app/mixins/discussion-editor-opengraph.js]: In the method above you check if contentImages exist before calling has images, shouldn't this happen here as well?

Comment (rogatty) [reply front/main/app/mixins/discussion-editor-opengraph.js]: It's not used anywhere else and is pretty straightforward.

**Pull request merged**